internal/template/resources.go

```diff
@@ -98,10 +98,13 @@ var supportedResourceTypes = []struct {
 		templateAbleNameFields: [][]string{
 			{"metadata", "name"},
 			{"spec", "application"},
-			// TODO: Somehow allow templating spec.params and spec.resolverRef.params
-			// which are arrays of name/value pairs. This would require changes to
+			// TODO: allow templating of spec.params
+			// which is an array of name/value pairs. This would require changes to
 			// applyResourceTemplate and possibly validateResourceNameFields
 		},
+		templateAbleFields: [][]string{
+			{"spec", "resolverRef", "params", "[]", "value"},
```

Comment:
perhaps lets allow templating names as well as values? 

we would need to check f the names need to mee the nameField requirements

Reply:
Was going for that initially, but was then thinking that those names in this case are fixed (url, revision, pathInRepo).
(`pathInRepo` already doesn't meey those requirements, as far as I understand)

I'm not sure it's worth trying to figure this out at this point.